<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Календарь Садовода</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelWelcome">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>911</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Garden Calendar</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelChoose">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>831</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Выберите пользователя</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="professionalButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>380</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Администратор</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/img/images/administator.png</normaloff>:/img/images/administator.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>170</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="background_start">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>10</y>
      <width>1321</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/img/images/background_start.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="amateurButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>380</y>
      <width>221</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: ; padding-bottom: 2px;</string>
    </property>
    <property name="text">
     <string>Садовод</string>
    </property>
    <property name="icon">
     <iconset resource="resourse.qrc">
      <normaloff>:/img/images/fermer.png</normaloff>:/img/images/fermer.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>170</width>
      <height>120</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelWelcome_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/img/images/pwenica.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelWelcome_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>40</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resourse.qrc">:/img/images/pwenicareverse.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <zorder>background_start</zorder>
   <zorder>labelWelcome</zorder>
   <zorder>labelChoose</zorder>
   <zorder>professionalButton</zorder>
   <zorder>amateurButton</zorder>
   <zorder>labelWelcome_2</zorder>
   <zorder>labelWelcome_3</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resourse.qrc"/>
 </resources>
 <connections/>
</ui>
